src/editor/editor.js

```diff
@@ -104,6 +104,19 @@ export default class Editor {
 		 */
 		this.t = this.locale.t;
 
+		/**
+		 * Indicates editor initialization status.
+		 *
+		 * The following statuses are available:
+		 * * initializing -  during initialization chain
+		 * * ready - after initializing chain, the editor is ready to work
```

Comment:
Relate to things which the user knows. When is the editor ready? When the promise returned by `create()` is resolved.